includes/Dashboard/Templates/Products.php

```diff
@@ -375,16 +371,12 @@ public function handle_product_update() {
             return;
         }
 
-        if ( ! isset( $_POST['dokan_update_product'] ) ) {
+        if ( ! isset( $_POST['dokan_update_product'] ) || ! wp_verify_nonce( sanitize_key( $_POST['dokan_edit_product_nonce'] ), 'dokan_edit_product' ) ) {
```

Comment:
here, we are checking 'dokan_update_product' and in wp_verify_nonce we are checking 'dokan_edit_product_nonce'. I think this is a mismatch.